### PR TITLE
Remove items that are covered by policy

### PR DESCRIPTION
**Pull request by fffej**
Remove items that are covered by [policy](https://info.red-gate.com/display/PD/Development+policies).

- [x] Are you happy for the content of this change to be publicly visible?
- [x] Are all new or updated entries marked as `isNew` = `true`?
- [x] Do `radar.csv` and `radar_libraries` render correctly when viewed on Github?
- [x] Does the updated [tech radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmove-policy%2Fradar.csv) display as you expect?
- [x] Does the updated [libraries tech radar](https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fraw.githubusercontent.com%2Fred-gate%2FTech-Radar%2Fmove-policy%2Fradar_libraries.csv) display as you expect?

### CODE REVIEW

**Comment by garethbragg** [comment]
The "Platforms" quadrant is pretty bare after this - what's that area useful for?

**Comment by garethbragg** [comment]
`radar.csv` doesn't actually render correctly at the moment. Thats broken on `master`, and should be fixed by #327

**Comment by fffej** [comment]
Re: The Platforms quadrant - I figure we pare back what's there to start with an then resolve that in a separate issue.

**Comment by TheEadie** [comment]
I agree with removing all these items as we should have policy documents that cover all of them. Do we know where each of these documents are on the info site? Could we link them from this PR so we know where to look int the future?

If we don't have a document / can't find it we should make sure there's an owner getting that sorted.

Policies I can see in here:
_Internal_
- The internal document storage policy

_Product_
- Windows platform support policy
- Web browser support policy
- SQL Server version policy
- SQL Server cloud / hosting environment policy
- SSMS version support policy

_Tech_
- API policy

File: radar.csv [hunk 1]
```diff
@@ -80,10 +52,7 @@ Babel,adopt,languages and frameworks,false,"Babel is a JavaScript compiler allow
...
 "Feature flags",adopt,techniques,false,"<a href='https://martinfowler.com/articles/feature-toggles.html'>A mechanism to prevent code in master from affecting the way the product behaves</a>. There are lots of different types of flags/toggles: they might be set at compile or run time; they might be purely internal or visible to the user, and so on."
-"ReSpeller using AmE",adopt,tools,false,"<a href='https://www.jetbrains.com/help/resharper/Spell_Checking.html'>ReSpeller</a> is a spell-checking engine integrated with ReSharper. We use American English to be consistent with .NET Framework and other 3rd party libraries. Additional words are stored in a solution team-shared settings file (*.sln.DotSettings)."
-"Open API specification",adopt,techniques,false,"Our public web-based APIs are exposed through the <a href='https://www.openapis.org/'>Open API</a> specification interface as being the description for REST APIs, which allows both humans and computers to discover and understand the capabilities of a service without requiring access to source code, additional documentation, or inspection of network traffic."
```
Comment (TheEadie) [inline radar.csv:84]: This is the only one I'm not sure about - We have the API policy that says everything should be documented - but we don't specify how. Should we have this on the radar as our preferred approach until it becomes policy?

Comment (fffej) [reply radar.csv:84]: I think the specification isn't an adoptable thing, we'd need to be more explicit about what to use.

Swagger is being explored by the Platform team, so I'll add that to Libraries?

**Pull request merged**